gcc.yaml

```diff
@@ -45,11 +45,22 @@ pipeline:
 
   - name: 'Configure GCC'
     runs: |
+      case "${{build.arch}}" in
```

Comment:
For future updates, can you add a comment here on how to identify the current march/mtune from cloud providers